baselines/dom.generated.d.ts

```diff
@@ -3465,6 +3466,9 @@ interface Element extends Node, GlobalEventHandlers, ElementTraversal, NodeSelec
     readonly scrollWidth: number;
     readonly tagName: string;
     innerHTML: string;
+    readonly assignedSlot: HTMLSlotElement | null;
```

Comment:
I initially wrote it like that, but I when I was doing some double-checking it looked like WebIDL's `?` suffix is different from typescript's `?`. Take a look at this and let me know what you think:
  * For turning an IDL specified type `T?` into an ES value, this sections seems like it's saying that only values from `T` and `null` are allowed: https://heycam.github.io/webidl/#idl-nullable-type
  * For turning an ES value into the IDL type `T?` then `undefined` and `null` are treated the same: https://heycam.github.io/webidl/#es-nullable-type

So for places where the DOM accepts an IDL type `T?` from ES code then `T | null | undefined` looks like the right TS type, but for readonly properties it seems like `T | null` is more correct.

Apologies if this debate has already been litigated and the decision to treat nullable values as also allowing undefined has been made across the board.

I have made the non-required members of dictionaries, like `EventInit#scoped` optional.